docs/ARCHITECTURE.md

```diff
@@ -1,17 +1,19 @@
 ## backpack-android architecture overview:
 
-`backpack-android` is a traditional Gradle Android lib project, composed of two modules.
+`backpack-android` is a traditional Gradle Android lib project, composed of the following modules.
```

Comment:
I don't think we ever show this/link to this doc - I just did a quick update, but probably worth looking into as part of the readme process if this should either live in the readme or be linked from there or is n't needed anymore